src/start/RealmScreen.js

```diff
@@ -74,21 +75,22 @@ class RealmScreen extends React.Component {
     const { progress, realm, error } = this.state;
 
     return (
-      <Screen title="Welcome" keyboardAvoiding>
+      <Screen title="Welcome to Zulip" keyboardAvoiding>
```

Comment:
Changing strings adds a lot of overhead to translating.
I don't see the point in this change.